### PR TITLE
Increase instances for distribution viewer script

### PR DESCRIPTION
**Pull request by robhudson**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-airflow/71)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: dags/longitudinal.py [hunk 1]
```diff
@@ -77,7 +77,7 @@
                       email=["telemetry-alerts@mozilla.com", "chudson@mozilla.com"],
                       execution_timeout=timedelta(hours=10),
                       release_label="emr-5.0.0",
-                      instance_count=5,
+                      instance_count=30,
```
Comment (vitillo) [inline dags/longitudinal.py]: I thought this wasn't needed any longer after stopping the SparkContext. Did I misunderstand?

Comment (vitillo) [inline dags/longitudinal.py]: Using many instances might cause to hit some AWS limits when multiple jobs are scheduled at the same time; let's try to not do that unless there is a good reason for it.

Comment (robhudson) [inline dags/longitudinal.py]: Yeah, this seems to be running fine as-is. 👍

**Pull request closed without merging**